src/app/drivingLicence/fields.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
       "required",
       { type: "maxlength", arguments: [43] },
```

Comment:
Should this be removed to avoid confusion?

Reply:
So I really want to remove this because I agree it's confusing but annoyingly it's needed, the maxLength validation syncs up with a form input feature in hmpo-form-wizard where even before form submission, a user will not be enter more than the amount of characters set in the arguments (as in the form loads on the page with a max length already set, in this case making it impossible for users to submit a surname longer than 43 characters)